tcf_website/templates/course/course.html

```diff
@@ -37,16 +37,10 @@ <h2>{{ course.title }}</h2>
             <br/>
 
             <ul class="instructor-list">
-                {% for i in recent_instructors %}
-                    <li class="instructor">
+                {% for i in instructors %}
+                    <li class="instructor {% if i.semester_last_taught != latest_semester %}old{% endif %}">
```

Comment:
Nice refactor!

Reply:
Actually, the idea of wrapping `old` in an `if` tag was taken from the department page. I'm just a copycat. 😄